THIRD-PARTY-NOTICES

```diff
@@ -13,10 +13,70 @@ See the License for the specific language governing permissions and
+  Commons Collections under The Apache Software License, Version 2.0
+  Commons Lang under The Apache Software License, Version 2.0
+  commons-beanutils under Apache License, Version 2.0
+  Default Plexus Container under The Apache Software License, Version 2.0
+  Digester under The Apache Software License, Version 2.0
+  dom4j under BSD
+  Doxia :: Core under Apache License, Version 2.0
+  Doxia :: Decoration Model under Apache License, Version 2.0
+  Doxia :: Logging API under Apache License, Version 2.0
+  Doxia :: Sink API under The Apache Software License, Version 2.0
+  Doxia :: Site Renderer under Apache License, Version 2.0
+  Doxia :: Skin Model under Apache License, Version 2.0
+  Doxia :: XHTML Module under Apache License, Version 2.0
+  JavaEWAH under Apache 2
+  javax.inject under The Apache Software License, Version 2.0
+  JGit - Core under Eclipse Distribution License (New BSD License)
+  JSch under Revised BSD
+  JSR-250 Common Annotations for the JavaTM Platform under COMMON DEVELOPMENT AND DISTRIBUTION LICENSE (CDDL) Version 1.0
+  JUnit under Common Public License Version 1.0
+  JZlib under Revised BSD
+  Maven Artifact under Apache License, Version 2.0
+  Maven Core under The Apache Software License, Version 2.0
+  Maven Model under Apache License, Version 2.0
+  Maven Model Builder under The Apache Software License, Version 2.0
+  Maven Plugin API under Apache License, Version 2.0
+  Maven Reporting API under The Apache Software License, Version 2.0
+  Maven Repository Metadata Model under The Apache Software License, Version 2.0
+  Maven Settings under The Apache Software License, Version 2.0
+  Maven Settings Builder under The Apache Software License, Version 2.0
+  org.eclipse.sisu.inject under Eclipse Public License, Version 1.0
+  org.eclipse.sisu.plexus under Eclipse Public License, Version 1.0
+  oro under Apache License, Version 2.0
+  Plexus :: Component Annotations under The Apache Software License, Version 2.0
+  Plexus Cipher: encryption/decryption Component under Apache Public License 2.0
+  Plexus Classworlds under The Apache Software License, Version 2.0
+  Plexus Common Utilities under Apache License, Version 2.0
+  Plexus I18N Component under Apache License, Version 2.0
+  Plexus Interpolation API under The Apache Software License, Version 2.0
+  Plexus Security Dispatcher Component under Apache Public License 2.0
+  Plexus Velocity Component under Apache License, Version 2.0
+  Sisu - Guice under The Apache Software License, Version 2.0
+  Sisu - Inject (JSR330 bean support) under The Apache Software License, Version 2.0
+  Sisu - Inject (Plexus bean support) under The Apache Software License, Version 2.0
+  SLF4J API Module under MIT License
+  sslext under Apache Software License, Version 1.1
+  Struts Core under The Apache Software License, Version 2.0
+  Struts Taglib under The Apache Software License, Version 2.0
+  Struts Tiles under The Apache Software License, Version 2.0
```

Comment:
Are you really using Struts in this project?

Reply:
ha, no, i need to check why that file didn't get updated.  I was thinking about writing a struts example (legacy migration use-cases), but stopped, i'll fix this

Reply:
wait... no... that isn't right i was thinking this was the `authn` repo. I'll check why that is getting pulled in.